src/components/icons/sm/Team.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
+    fill="currentColor"
+    d="M10.286 5.444c0 1.35-1.023 2.445-2.285 2.445-1.263 0-2.286-1.095-2.286-2.445C5.715 4.094 6.738 3 8 3c1.262 0 2.285 1.094 2.285 2.444ZM3.428 11.959c0-.986.552-1.88 1.43-2.182.937-.323 2.164-.668 3.141-.668.977 0 2.205.345 3.142.668.877.302 1.43 1.196 1.43 2.182v.817c0 .675-.512 1.222-1.143 1.222H4.57c-.632 0-1.143-.547-1.143-1.222v-.817Z"/><path fill="#4D586B" fill-rule="evenodd" d="M4.633 4.751a3.934 3.934 0 0 0 .264 2.253c-.3.53-.846.885-1.468.885-.947 0-1.714-.82-1.714-1.833s.767-1.833 1.714-1.833c.47 0 .895.201 1.204.528Z" clip-rule="evenodd"/><path fill="#4D586B" d="M2.286 12.776v-.817c0-1.108.474-2.181 1.31-2.845a3.429 3.429 0 0 0-.167-.005c-.733 0-1.654.259-2.357.501C.415 9.837 0 10.507 0 11.246v.613c0 .507.384.917.857.917h1.429Z"/><path fill="#4D586B" fill-rule="evenodd" d="M11.372 4.751a3.937 3.937 0 0 1-.263 2.253c.3.53.845.885 1.466.885.946 0 1.713-.82 1.713-1.833s-.767-1.833-1.713-1.833c-.468 0-.893.201-1.203.528Z" clip-rule="evenodd"/><path fill="#4D586B" d="M13.716 12.776v-.817c0-1.108-.474-2.181-1.31-2.845.057-.003.113-.005.168-.005.732 0 1.652.259 2.354.501.657.227 1.071.897 1.071 1.636v.613c0 .507-.383.917-.856.917h-1.427Z" />
+</svg>
```

Comment:
is it correct that only the first `<path>` has `currentColor`? The other 4 have `#4D586B`

Reply:
5eca0982ffe131fa069d92f7dc20598ed10cc66d

Reply:
it's not the same. @michalarchilogic is it ok if all 3 people are "black", instead of the center person being "black" and the left and right person "gray"? This way it's easier to be able to change the color (e.g. on hover, etc.)

![Screenshot 2023-07-18 at 18 28 43](https://github.com/archilogic-com/honeycomb/assets/89903103/00534055-458f-4c93-aeeb-eb87f2e9199d)

Reply:
Wait, what do you mean by black? They all should be grey:
<img width="674" alt="image" src="https://github.com/archilogic-com/honeycomb/assets/7584821/46167bd1-6f4b-40bc-adf2-479994a366cb">

Reply:
or do you mean a default grey color? The storybook from this PR:
<img width="283" alt="image" src="https://github.com/archilogic-com/honeycomb/assets/7584821/e1e7b759-7843-461f-9ab4-259f1a75c6e4">

Reply:
basically, when adding a new icon, we need to be able to answer the question: how is this icon reacting to color changes https://honeycomb.archilogic.com/?path=/docs/components-icons--change-color? 

possible answers:
- not. It's always the same color. Then there should be no `currentColor` in the SVG
- some parts of the icon change color. Those parts should have `currentColor` in the SVG
- all of the icon changes color. All colors must be `currentColor` in the SVG

Reply:
that's what I meant in the README https://github.com/archilogic-com/honeycomb/blob/main/README.md#adding-icons with 

> replace (some) colors with currentColor so the color of the icon can be set via CSS

but maybe the instructions could be improved...

Reply:
If we already use the default grey color for icons with `currentColor` value what is the reason to hardcode the same grey color by HEX code for icons without reacting to color changes?

Reply:
I don't understand the question. Do what you want :)

Reply:
😄 
> not. It's always the same color. Then there should be no currentColor in the SVG

By that, do you mean to copy-paste `#4D586B` from Figma and replace all the `currentColor`? Because there are no hovers/color modifications for this "Team" icon?

Reply:
> it's not the same. @michalarchilogic is it ok if all 3 people are "black", instead of the center person being "black" and the left and right person "gray"? This way it's easier to be able to change the color (e.g. on hover, etc.)
> 
> ![Screenshot 2023-07-18 at 18 28 43](https://user-images.githubusercontent.com/89903103/254331569-00534055-458f-4c93-aeeb-eb87f2e9199d.png)

The icon should always have only one fill color: "Stone" or "Navy" (if it's a button as well). I don't know where having both gray and black fills together comes from. If it's somewhere in the designs it's my bad. Point me to it and I'll fix it.